src/js/background/messageHandler.js

```diff
@@ -23,6 +23,9 @@ const messageHandler = {
       case "resetBookmarksContext":
         response = assignManager.resetBookmarksMenuItem();
         break;
+      case "resetProxySupport":
+        response = assignManager.init();
```

Comment:
Is this the right UX?

1. Enable proxy support
2. Assign Container1 to use proxy1
3. Open sensitiveSite1 in Container1 (thru proxy1)
4. Disable proxy support
5. Reload tab from step 3
   * sensitiveSite1 is reloaded without a proxy without any other warning